test/bot-test.rb

```diff
@@ -0,0 +1,31 @@
+# - Install dependencies
+#   $ gem install readline
+#   $ gem install nats
+#
+# - Start a NATS server
+#   $ nats-server
+#
+# - Start the bot
+#   $ make macondo_bot && ./macondo_bot
+#
+# - Start the test shell
+#   $ ruby test/bot-test.rb
+
+require 'nats/client'
+require 'fiber'
+require 'readline'
+
+CHAN = 'macondo.bot'
+
+NATS.start do
```

Comment:
fwiw Go has great NATS integration as well, you can just include it as a library since it's written in Go, but this is fine

Reply:
yeah, i'll rewrite it in go at some point, i just found it more convenient to use ruby while figuring out the nats protocol.